chore: Update all references of 'Twitter' to 'X.com'

NikolaiKryshnev opened the pull request:
- Replaced mentions of "Twitter" with "X.com" across multiple files to align with the rebranding.
- Updated links, descriptions, and any related text where applicable.

cb-heimdall commented:
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/2</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>1</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>2</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>
<!-- CODEOWNERS & Bar Raisers -->
<!-- Power Rangers -->
<!-- Links -->

wbnns commented:
@NikolaiKryshnev 

Hello, thanks for contributing to Base! There are currently merge conflicts as well as several changes that are unrelated to changing the references of Twitter to X. Please resubmit a new PR with only the reference updates.

NikolaiKryshnev commented:
@wbnns Hello!

Thank you for the feedback. I have addressed the issues by creating a new pull request with only the updates to replace references to Twitter with X. Here is the link to the new PR: [https://github.com/base-org/web/pull/1353]).

Please let me know if any further adjustments are needed.
